docs/generators/scala-play-server.md

```diff
@@ -83,7 +83,6 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 <li>case</li>
 <li>catch</li>
 <li>class</li>
```

Comment:
I think we dont't need change doc for other languages here.

Reply:
Change reverted and updated the required file only